components/Icon/assets/video-chat.svg

```diff
@@ -0,0 +1,5 @@
+<!-- Generated by IcoMoon.io -->
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24">
```

Comment:
yes, the defaults are 24 & 24 (answering my previous comment about default sizes)